src/app/scenarios/scenarios-list/scenario-list.component.test.js

```diff
@@ -30,25 +51,48 @@ describe('ScenarioListComponent', () => {
         });
 
         it('should get scenarios', () => {
-            expect(scenarioService.getScenarios).toHaveBeenCalled();
+            expect(scenarioService.get).toHaveBeenCalled();
         });
+    });
 
-        it('should resolve promised list of scenarios', (done) => {
-            scenarioService.getScenarios().then(() => {
-                expect(sut.scenarioList).toEqual(listData);
+    describe('#headers', () => {
+        const allowedHeaders = [
+            { topic: 'name', name: 'Name', sortable: true },
+            { topic: 'active', name: 'Active', sortable: false },
+            { topic: 'description', name: 'description', sortable: true },
+        ];
 
-                done();
-            });
+        it('should have collection of allowed headers: name, active, description', () => {
+            sut._headers = allowedHeaders;
+            expect(sut.headers).toEqual(allowedHeaders);
         });
     });
 
-    describe('Open scenario', () => {
-        it('will open scenario by id', () => {
-            const id = 1;
-            const scenario = {id};
+    describe('#removeScenario', () => {
+        it('should be defined', () => {
```

Comment:
this test is useless cause next test will fail in any case if there will be no "removeScenario" method. !fix